analyzers/src/SonarAnalyzer.Common/Common/FixAllProviders/DocumentBasedFixAllProvider.cs

```diff
@@ -48,16 +48,16 @@ private static string GetFixAllTitle(FixAllContext fixAllContext)
             switch (fixAllContext.Scope)
             {
                 case FixAllScope.Document:
-                    return string.Format(titleScopePattern, diagnosticId, fixAllContext.Document.Name);
+                    return string.Format(TitleScopePattern, diagnosticId, fixAllContext.Document.Name);
```

Comment:
Can you "cover as you code" here? We have some significant code paths without any coverage here :/

The same UT should then also cover the missing logic below.